test/integration/openshift_health_checker/setup_container.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Required vars:
+# * scenario = unique name for the container to be started
+# * image = name of the image to start in the container
+# Optional vars:
+# * l_groups = host groups that the container should be added to
```

Comment:
`groups` and `host_vars` are both already used by ansible. `l_` doesn't have any official meaning but I've seen it a lot in our ansible code to mean something that's "local" in some sense.